src/masternode.cpp

```diff
@@ -530,7 +512,6 @@ bool CMasternodeBroadcast::CheckAndUpdate(int& nDos)
         //take the newest entry
         LogPrint(BCLog::MASTERNODE,"mnb - Got updated entry for %s\n", vin.prevout.hash.ToString());
         if (pmn->UpdateFromNewBroadcast((*this))) {
-            pmn->Check();
             if (pmn->IsEnabled()) Relay();
```

Comment:
for a future PR:
this `IsEnabled` is redundant now. It's being checked few lines above.

Reply:
Good catch. Makes sense to remove it here.